go/install/stop_windows.go

```diff
@@ -0,0 +1,20 @@
+package install
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/keybase/client/go/libkb"
+)
+
+func StopAllButService(mctx libkb.MetaContext) {
```

Comment:
does this function actually do anything?  seems a bit like a noop

Reply:
apparently opening that file causes kbfsdokan to unmount

Reply:
That's right, I remember this